timed/redmine/management/commands/update_project_expenditure.py

```diff
@@ -65,11 +76,18 @@ def handle(self, *args, **options):
                     "value": project.amount_invoiced.amount,
                 },
             ]
-            try:
-                issue.save()
-            except redminelib.exceptions.BaseRedmineError as e:  # pragma: no cover
-                self.stdout.write(
-                    self.style.ERROR(
-                        f"Failed to save Project {project.name} with Redmine issue {issue.id}!\n{e}"
+            if not pretend:
+                try:
+                    issue.save()
+                except redminelib.exceptions.BaseRedmineError as e:  # pragma: no cover
+                    self.stdout.write(
+                        self.style.ERROR(
+                            f"Failed to save Project {project.name} with Redmine issue {issue.id}!\n{e}"
+                        )
                     )
+
```

Comment:
`continue` missing?

If this is not pretending and throws an exception, it will both write an error and success message, won't it?

Reply:
Yep, you're right. Thanks :)